db/upgrade_migrations/20160723012349_upgrade_v062_to_v063.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class UpgradeV062ToV063 < ActiveRecord::Migration[5.0]
+  def up
+    i = 1
+    Thredded::MessageboardGroup.order(created_at: :desc).each do |group|
+      Rails.logger.info group.id
+      next if group.valid?
+
+      group.name = "#{group.name}-#{i}"
+      group.save!
+      i += 1
+    end
```

Comment:
Is there a way to do this with plain SQL? General rule of thumb within migrations is to avoid using AR models in the off chance they go away, or their structure changes.